kolibri/plugins/setup_wizard/tasks.py

```diff
@@ -25,6 +26,16 @@
 from kolibri.core.tasks.decorators import register_task
 
 
+class IsListingJobs(BasePermission):
+    """
+    When a subset_of_users_device is provisioned, after the first user has been synced,
+    listing of COMPLETED jobs must be allowed for the anonymous user that's making the request
+    """
+
+    def has_permission(self, request, view):
+        return view.action == "list"
+
```

Comment:
Should we check that the device is indeed `subset_of_users_device` and `is_provisioned` and then go ahead with giving access on `view.action == "list"`...?

Reply:
Umm, gotcha!